addons/web/static/tests/search/search_bar.test.js

```diff
@@ -1379,7 +1389,6 @@ test("edit a field", async () => {
     expect(`.o_searchview_facet.o_facet_with_domain .o_searchview_facet_label`).toHaveCount(1);
 
     await editSearch("def");
-    keyDown("ArrowDown");
```

Comment:
This was wrong, as the first item of the list is already selected. But there was only one item so it went unnoticed. 
With the "Add custom filter", it would go down on it and fail the test.